### DISCUSSION TITLE
Gardening in Upstate New York

### DISCUSSION THREAD

**Discussion by spiritedgoldfish** [katskillskat/katskills-org#8] [Q&A]
If there was a way to eat rocks, we would be set. Gardening is one of the many hobbies people hold here, however, our placement on the map only gives us a 108-day growing season on average. A growing season this short limits what you can grow, when you can grow, and how you grow it. Some crops can't be grown here at all due to time restraints. It can be overwhelming to begin a new hobby and find that the elements and odds are stacked against your success. In starting a garden, I've spent a fair amount of time down the rabbit hole that is Youtube. As of yet, I haven't found a channel producing content in our specific gardening zone. I attribute this to the fact that a content stream from someone gardening in our areas would be incredibly limited and less than profitable to the creator. In the last few years, I have definitely noticed a resurgence of the hobby as higher-than-average food insecurity raked through with Covid-19. 
What are your gardening tips and tricks? Was there something you learned from a parent or grandparent that has changed how you keep your own garden?

**Comment by kendra-lacey** [comment]
I have nothing to contribute to this as I am not a green thumb but I did recently come across this local gardening group 
Instagram- @halcottcommunitygardeb
Web- hacottgardeners.blogspot.com
I know they are having an event in June that might be a good resource to answer some questions
